src/hubinit/failedcontentpull.bal

```diff
@@ -0,0 +1,51 @@
+import ballerina/http;
+import mosip/repository;
+import ballerina/lang.'int;
+import ballerina/config;
+
+@http:ServiceConfig {
+    basePath: "/websub/hub"
+}
+service failedcontentpull on hubListener {
+
+    resource function failedmessage(http:Caller caller,
```

Comment:
Can this be camel case as per Ballerina coding standards?

Reply:
yes